build.sbt

```diff
@@ -165,7 +165,7 @@ lazy val serviceDiscoveryLagom13Java = createProject("reactive-lib-service-disco
       "com.lightbend.lagom" %% "lagom-javadsl-client" % Versions.lagom13 % "provided"
     ),
     assemblyInclude(),
-    assemblyExcludeLocal("common", "servicediscovery")
+    assemblyExcludeLocal(Seq("common"), Seq("servicediscovery", "javadsl"), Seq("servicediscovery", "scaladsl"))
```

Comment:
This wackiness is because we don't want to include `com.lightbend.servicediscovery.javadsl` in the jar, but we do want to include `com.lightbend.servicediscovery.lagom.javadsl`.